ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -400,6 +402,145 @@ def check_alba(logger):
                            'arakoon_connected')
 
     @staticmethod
+    @ExposeToCli('alba', 'disk-safety')
+    def get_disk_safety(logger):
+        """
+        Send disk safety for each vpool and the amount of namespaces with the lowest disk safety to DB
+        """
+        points = []
+        abms = []
+
+        test_name = 'disk-safety'
+        result = {
+                "repair_percentage": None,
+                "lost_disks": None
+            }
+        for service in ServiceHelper.get_services():
+            if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR:
+                abms.append(service.name)
+
+        abms = list(set(abms))
```

Comment:
Was to only have unique values.
However the check can happen when appending to abms in the first place.